src/utils/py/daos_api.py

```diff
@@ -1893,13 +1895,15 @@ def list(self, coh):
                                .format(retcode))
         return epoch.value
 
-    def open(self, coh):
+    def open(self, coh, epoch=None):
```

Comment:
If the intent is really to open a snapshot, then the epoch should be mandatory there.